_posts/2016-9-2-jade-release.markdown

```diff
@@ -0,0 +1,67 @@
+Hello everyone,
```

Comment:
This should be removed, and the next sentence should be reworded to sound more like a blog post and not an email